src/studio/src/designer/frontend/packages/schema-editor/test/__tests__/__snapshots__/SchemaInspector.test.tsx.snap

```diff
@@ -85,7 +85,7 @@ exports[`Should match snapshot (enums): enums 1`] = `
           <input
             aria-invalid="false"
             class="MuiInputBase-input MuiInput-input"
-            id="definitionsStatistiskeEnhetstyper-name"
+            id="selectedItemName"
```

Comment:
This indicates an issue where we do not have uniqueness of ids.
It is very important, and we should have tests on our snapshots if we don't have that already.

Reply:
Again, there is just one instance of this textfield, so the id will be unique